### PR TITLE
Update dependencies with test fixes

### PR DESCRIPTION
**Pull request by moczolaszlo**
Hello

I updated the dependecies and fixed the failed tests.
💻

### CODE REVIEW

File: Gruntfile.js [hunk 1]
```diff
@@ -46,8 +46,8 @@ module.exports = function(grunt) {
           file: 'test/fixtures/_component.json',
           changelog: 'test/fixtures/_CHANGELOG.md',
           additionalFiles: ['test/fixtures/_bower.json'],
-          changelogText: '### <%= version %>\n',
-          commitMessage: 'v<%= version %>'
+          changelogText: grunt.template.process('### <%= version %>\n', {data: {'version': '0.0.13'}}),
```
Comment (dorgan) [inline Gruntfile.js]: This will cause a problem as it adds a requirement that we update the Gruntfile every time we update the version

Comment (moczolaszlo) [inline Gruntfile.js]: Why? In the /test/fixtures/component.json file the version will be always 0.0.12.

Comment (dorgan) [inline Gruntfile.js]: hmm maybe your right, going to look back in history at the tests...

**Comment by dorgan** [comment]
@drublic I believe the change is safe, any opinion?

**Comment by moczolaszlo** [comment]
Any update?

**Pull request merged**